src/mixs/schema/mixs.yaml

```diff
@@ -3936,6 +3936,8 @@ slots:
       Preferred_unit: degree Celsius
     description: Temperature of the air at the time of sampling
     title: air temperature
+    examples:
+      - value: 25 degree Celsius
```

Comment:
Excellent! That unit validates against https://ucum.nlm.nih.gov/ucum-lhc/demo.html!

Reply:
Sorry, I have to withdraw that. The UCUM unit for degrees Celsius is just `Cel`

Now, whether GSC want' to constrain our preferred units to the UCUM guidelines is another question.

Reply:
Which is

```yaml
  air_temp:
    annotations:
      Preferred_unit: degree Celsius
    description: Temperature of the air at the time of sampling
    title: air temperature
    keywords:
      - air
      - temperature
    slot_uri: MIXS:0000124
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
```

Reply:
I think the use of UCUM units is an important future discussion but shouldn't hold this PR up, since it follows the preferred units guidelines, what was extracted from an GSC spreadsheet.